### DISCUSSION TITLE
Декоративный элемент прогресс бар

### DISCUSSION THREAD

**Discussion by sashagm** [moonshine-software/moonshine#237] [Ideas]
Предлагаю добавить отдельный декор элемент прогресс бар или расширить стилями. 

Сейчас есть эта возможность но выводится цифрами, можно сделать как bootstrap ввиде виджета https://getbootstrap.com/docs/5.3/components/progress/  

ValueMetric::make('Open tasks')
            ->value(Task::opened()->count())
            ->progress(200) // Конечная цель

**Comment by lee-to** [comment]
Добавил в задачи
